omc3/harpy/clean.py

```diff
@@ -32,7 +32,7 @@ def clean(harpy_input, bpm_data, model):
     """
     bpm_data, bpms_not_in_model = _get_only_model_bpms(bpm_data, model)
     if bpm_data.empty:
-        raise AssertionError("Check BPMs names! None of the BPMs was found in the model!")
+        raise AssertionError("Check BPMs names! None of the BPMs were found in the model!")
```

Comment:
This is wrong, change it back.

Reply:
https://data.grammarbook.com/blog/singular-vs-plural/none-were-vs-none-was/

Reply:
This exists only in US english.